packages/cypress/src/index.ts

```
@@ -0,0 +1,73 @@
+export function getChaosScript(): string {
+  return scriptContent;
+}
+
+/**
+ * Get the chaos script with configuration injected
+ */
+export function getChaosScriptWithConfig(config: ChaosConfig): string {
+  try {
+    const serializedConfig = JSON.stringify(config).replace(/</g, '\\u003C');
+    return `window.__CHAOS_CONFIG__ = ${serializedConfig}; ${scriptContent}`;
+  } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)